vendor/k8s.io/kubernetes/pkg/kubectl/resource_printer.go

```diff
@@ -844,39 +868,8 @@ func printReplicaSetList(list *extensions.ReplicaSetList, w io.Writer, options P
 	return nil
 }
 
-func printCluster(c *federation.Cluster, w io.Writer, options PrintOptions) error {
```

Comment:
This removal along with other changes to "cluster", looks suspicious.
@juanvallejo could you please confirm this is intentional? Why is it being removed?